beesdoo_website_shift/views/my_shift_website_templates.xml

```diff
@@ -14,6 +14,17 @@
         </xpath>
     </template>
 
+    <!-- action button to shift swap -->
+    <record
+        model="ir.actions.act_window"
+        id="action_button_shift_swap">
+        <field name="name">Swap Shift</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">beesdoo.shift.subscribed_underpopulated_shift</field>
+        <field name="view_type">tree</field>
+        <field name="view_id" ref="beesdoo_shift_swap.shift_swap_view_tree"/>
+    </record>
+
```

Comment:
I think that this should be defined in module `beesdoo_website_shift_swap`.

Reply:
right, I did defined in module beesdoo_website_shift_swap. Just forgot to delete it here.